### ISSUE TITLE
Which accessibility testing tool should you use?

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#177]
### Overview
Figure out which accessibility tool is right for testing the accessibility of the newly added project status indicator and indicate your recommendation in this issue.

There are more than 100 accessibility testing tools.  Figuring out which ones to use can be a black hole.  For guidance we recommend this article:  [Which accessibility testing tool should you use?](https://medium.com/pulsar/which-accessibility-testing-tool-should-you-use-e5990e6ef0a)

### Summary of Article
The author recommends using the tools in the following order fixing as you go along, since no one tool catches all the relevant issues

[aXe](https://www.deque.com/axe/)
[SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)
[Tenon](https://tenon.io/)
[WAVE](https://wave.webaim.org/extension/)
[Lighthouse](https://developers.google.com/web/tools/lighthouse/#devtools)

But if you want to test your site with other tools, here is a bigger [list](https://www.w3.org/WAI/ER/tools/)

**Comment by KianBadie** [comment]
Thank you for the starting resources for search of what accessibility tool to use Bonnie! I've only done a little bit of research, but it definitely does seem like a little bit of a rabbit hole trying to find "the ones" to use. 

As a starting point, using a combination of the trio of aXe, SiteImprove, and WAVE sound ok to me. Like they would be able to pick up at least low hanging fruit? I will try to do more research, but I also want to type this out to prevent myself of endlessly going down the rabbit hole. 

I think I have been seeing WAVE and aXe around though. From the article you provided, it seemed like aXe and SiteImprove had some thorough testing, which was appealing for me. I'll try to play around with the 3 on our site and see what my experience is. 

AS for using these tools, there were command lines tools (which I think was used for automated testing) and browser extensions. I was going to use browser extensions, but was automated accessibility testing something you were interested in? Sorry if that question isn't super clear, I still don't fully understand automated accessibility testing.

**Comment by ExperimentsInHonesty** [comment]
This issue is template I built to introduce all developers to the concept if useability testing.  I think you have the concept now.  So, its time to just start experimenting and as you make pr requests, if they are going to change the look of the site, run whatever accessabilty tools you think are appropriate and mention which in your pull requests so others can learn from reading the closed issues.

**Comment by KianBadie** [comment]
Got it, makes sense, I will be doing that from now on then!

**Comment by ExperimentsInHonesty** [comment]
@harishlingam you can close this when you have finished reading.

**Comment by ExperimentsInHonesty** [comment]
@harishlingam can we close this issue now?
